index.html

```diff
@@ -180,6 +180,23 @@ <h3>Skills</h3>
             </ul>
         </section>
     </section>
+    <section class="last">
+        <div class="footer">
+        <h1 class="contact-title">Contact me</h1>
+        <p class="contact-description">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <br>
+        <form class="contact-form">
```

Comment:
- [ ] Please delete the outline and add padding for inputs: 

![image](https://user-images.githubusercontent.com/77038610/150155929-0f1b7243-5b65-4b14-beae-1a9871245114.png)
You can easily delete the outline by setting `outline: none`